kani-driver/src/metadata.rs

```diff
@@ -151,13 +151,28 @@ impl KaniSession {
     }
 }
 
+/// Sort harnesses such that for two harnesses in the same file, it is guaranteed that later
+/// appearing harnesses get processed earlier.
+/// This is necessary for the concrete playback feature (with in-place unit test modification)
+/// because it guarantees that injected unit tests will not change the location of to-be-processed harnesses.
+pub fn sort_harnesses(harnesses: &[HarnessMetadata]) -> Vec<HarnessMetadata> {
```

Comment:
nit: `sort_by_location` or something like that since correctness depend on what we are sorting it by.

Reply:
Done. Updated func name.

Reply:
Thanks